pkg/Library/HuggingFace/HuggingFaceNodeTypes.js

```diff
@@ -9,7 +9,7 @@ export const HuggingFaceNodeTypes = {
   HuggingFaceImage: {
     category,
     description: 'Generates an image using one of the HuggingFace models',
-    type: '$library/HuggingFace/Nodes/HuggingFaceImageNode',
+    type: '$anewLibrary/HuggingFace/Nodes/HuggingFaceImageNode',
```

Comment:
this is not `anew` file (and some others around it), did you intend to edit all of them?